packages/client/lib/Nft.ts

```diff
@@ -0,0 +1,60 @@
+import { NftProvider, Address, Transaction } from '@liquality/types'
+
+export default class Nft implements NftProvider {
+  client: any
+
+  constructor(client: any) {
+    this.client = client
+  }
+
+  /** @inheritdoc */
+  async balance(contract: Address | string, tokenIDs?: number | number[]): Promise<number | number[]> {
+    const balance = await this.client.getMethod('balance')(contract, tokenIDs)
+    return balance
+  }
+
+  /** @inheritdoc */
+  async transfer(
+    contract: Address | string,
+    receiver: Address | string,
+    tokenIDs: number | number[],
```

Comment:
If we are not going to separate to transfer-transferBatch function, we should atleast make this function always take tokenIDs as array and in the nft standard that can't send batch: throw an error when the list contains more than one